packages/fuels-core/src/types/bits_256.rs

```diff
@@ -50,19 +50,80 @@ impl Tokenizable for Bits256 {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+// ANCHOR: evm_address
+pub struct EvmAddress {
+    value: Bits256,
```

Comment:
I think it should be documented "in-code" that the first twelve bytes are zeroed out

Reply:
How would you go about this?

Reply:
So it would be comment duplication but a comment along the line of the one you wrote on the `clear_bytes` function would be good enough imo. It's just so people don't get confused.